x-pack/plugin/esql/src/main/generated/org/elasticsearch/xpack/esql/expression/function/scalar/ip/IpPrefixEvaluator.java

```diff
@@ -42,12 +42,12 @@ public IpPrefixEvaluator(Source source, EvalOperator.ExpressionEvaluator ip,
       EvalOperator.ExpressionEvaluator prefixLengthV4,
       EvalOperator.ExpressionEvaluator prefixLengthV6, BytesRef scratch,
       DriverContext driverContext) {
-    this.warnings = new Warnings(source);
```

Comment:
This is unrelated to my change, but got generated due to a recent change. Let me know if I should remove it.

Reply:
Nah, it's fine. It's got to get in eventually. It's just two PRs passing each other in the night, unseen.